tools/wini/wini.py

```diff
@@ -8,6 +8,9 @@
 
 from . import print, subprocess, workspace
 
+_CORE_WHEEL_DIR = pathlib.Path("./.wini/core/")
```

Comment:
will this work on win? i'm thinking when we use wini to build the wheels (on windows) for a release.